tgui/packages/tgui/interfaces/CommunicationsConsole.js

```diff
@@ -382,15 +384,26 @@ const PageMain = (props, context) => {
           )}
           {!!canToggleEmergencyAccess && (
             <Button.Confirm
-              icon="id-card-o"
+              icon="wrench" // SKYRAT EDIT
```

Comment:
Leave this one as it is and put the wrench on the engineering one, that makes a lot more sense.
```suggestion
              icon="id-card-o"
```